server.go

```diff
@@ -12,9 +12,6 @@ import (
 	"time"
 )
 
-// Maximum number of TCP queries before we close the socket.
-const maxTCPQueries = 128
```

Comment:
We should expose this in the server, and make `0` mean; disable this. 

By removing this the server has no way to kill a client after it things it has enough; Google Public DNS does the same thing btw, so it's not uncommon to have this.